### PR TITLE
Addition of Symbol in JP version

### PR DESCRIPTION
**Pull request by Laioxy**
Each symbol confirmed in JP version is added.

Also, the description of UNOWN_STONE_DROP_CHANCE was different from the actual value. (21% -> 33%)
The binary "0x21" was probably mistaken for "21%".

---
Nice to meet you. I am a Japanese who is analyzing PMD2 Sky.
I sent a pull request for the first time to contribute to the JP ROM support of SkyTemple.
For now, I have added only the parts that I can confirm, filling in the holes. Please check it out.

(My English is terrible, so I use DeepL translation. Sorry if the text is incomplete)

### CODE REVIEW

**Comment by theCapypara** [comment]
:eye: :eye: 
This may close skytemple/skytemple-files#235 - does this PR contain all of these symbols?
In any case I'll test it later and add you to the SkyTemple credits.

**Comment by theCapypara** [comment]
Thank you!

**Comment by Laioxy** [comment]
Unfortunately, I have not been able to find all the symbols as I am a newbie to this analysis and have only been doing it for a year.
As soon as I find them, I will add them!

File: symbols/arm9.yml [hunk 1]
```diff
@@ -9005,32 +9051,42 @@ arm9:
...
         NA: 0x20A18EC
+        JP: 0x20A2CC0
       length:
         NA: 0x4
+        JP: 0x4
       description: "The prefactor to the output of the ClampedLn in the damage formula, as a binary fixed-point number with 8 fraction bits (50)."
-    - name: DAMAGE_FORMULA_AT_PREFACTOR
+    - name: DAMAGE_FORMULA_DEF_PREFACTOR
       address:
         NA: 0x20A18F4
```
Comment (UsernameFodder) [inline symbols/arm9.yml]: Good catch, this was actually a mistake. The NA address here should be `0x20A18F0`. I'll update it.

File: symbols/overlay10.yml [hunk 2]
```diff
@@ -242,104 +268,133 @@ overlay10:
         EU: 0x2
         NA: 0x2
         JP: 0x2
-      description: "The chance of an Unown dropping an Unown stone, as a percentage (21%)."
```
Comment (UsernameFodder) [inline symbols/overlay10.yml:245]: This was probably my mistake 😅. That's embarrassing.

**Review by UsernameFodder — APPROVED** [review]
Everything looks good here. Thanks for doing all this work!

**Pull request merged**